### DISCUSSION TITLE
A medal and scoreboard system

### DISCUSSION THREAD

**Discussion by dannyaguero** [NovaoTech/solaris#11] [Ideas]
Similar to the Newgrounds.io API, Can there be a scoreboard system where people can use post high scores, and a medal system where people can get points? Maybe a custom addon since Turbowarp is being used? (at least as i read in [https://github.com/JTechnologies/Polaris/discussions/5#discussioncomment-986241](url) <--- this post)

**Comment by MikeDev101** [comment]
I could provide backend support as a part of CloudDisk, but I like this idea.
